rust/rama/Cargo.toml

```
@@ -0,0 +1,20 @@
+[package]
+name = "server"
+version = "0.0.0"
+edition = "2021"
+publish = false
+
+[profile.release]
+opt-level = 3
+debug = false
+debug-assertions = false
+lto = true
+panic = "abort"
+incremental = false
+codegen-units = 1
+rpath = false
+strip = false
+
+[dependencies]
+rama = "0.2.0-alpha.6"
```

```suggestion
rama = { version = "0.2.0-alpha.6", features = ["http-full"] }
serde = "1.0"
```

you'll need http-full feature enabled to get this working